indra/newview/linux_tools/refresh_desktop_app_entry.sh

```diff
@@ -30,20 +30,39 @@ install_desktop_entry()
 
     desktop_entry="\
 [Desktop Entry]\n\
+Version=1.4\n\
 Name=${launcher_name}\n\
+GenericName=Second Life Viewer\n\
 Comment=Client for the On-line Virtual World, Second Life\n\
 Exec=${installation_prefix}/alchemy\n\
-Icon=${installation_prefix}/alchemy_icon.png\n\
+Icon=${desktopfilename}\n\
```

Comment:
Since `xdg-icon-resource` is being used, is this line necessary? I have not researched this, but am curious about the change here.

Reply:
Yes - because `xdg-icon-resource` isn't intrinsically linked to the desktop file, we need to tell it the name of the icon to use in the icons repo.